### DISCUSSION TITLE
Whether the code in the env_stocktrading_np file is faulty？

### DISCUSSION THREAD

**Discussion by dd-python** [AI4Finance-Foundation/FinRL#993] [Q&A]
When tested with the env_stocktrading_np environment, the action to sell the stock does not take into account whether the stock is currently held.

**Comment by abdulazeezb** [comment]
Are you able to run the env_stocktrading_np  class without any modifications,?

I have to copy the class and comment on the lines starting with 'logger' even after that the prediction methods don't work properly 😕
